src/sentry/lang/native/minidump.py

```diff
@@ -180,3 +194,73 @@ def frames_from_minidump_thread(thread):
         'package': frame.module.code_file if frame.module else None,
         'trust': frame.trust,
     } for frame in reversed(list(thread.frames()))]
+
+
+def get_attached_minidump(data):
+    cache_key = cache_key_for_event(data)
+    attachments = attachment_cache.get(cache_key) or []
+    return next((a for a in attachments if a.type == MINIDUMP_ATTACHMENT_TYPE), None)
+
+
+def merge_symbolicator_minidump_response(data, response):
+    sdk_info = get_sdk_from_event(data)
+
+    # TODO(markus): Add OS context here when `merge_process_state_event` is no
+    # longer called for symbolicator projects
+
+    data.setdefault('debug_meta', {})['images'] = images = []
+
+    for complete_image in response['modules']:
```

Comment:
Note to self: As opposed to other properties, symbolicator always outputs `modules` and `stacktraces`, even if they are empty. We need to make sure that this is explicit (i.e. documented).